docs/developer-docs/docs/operate-a-node/barra-testnet/join-barra.md

```
@@ -0,0 +1,160 @@
+  ```
+- `client.toml`   
+  Set the chain ID:
+  ```bash
+  chain-id = "barra_9191-1"
+  ```
+- `config.toml`   
+  Add [seed nodes](https://github.com/warden-protocol/networks/tree/main/testnets/barra/chain.json), enable/disable the mempool:
+  ```bash
+  seeds = "c489c003b7c72298840bd4411ffc98ce13e07c27@54.194.136.183:26656,4564c91423a923eaba7982e69e33aec6185d362f@54.72.5.234:26656"
+  ```
+  ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)